src/pages/billing/NewBillingProjectWizard/AzureBillingProjectWizard/AddUsersStep.test.ts

```diff
@@ -20,6 +20,7 @@ export const addUserAndOwner = (userEmail, ownerEmail) => {
 };
 
 const defaultProps = {
+  inputDebounce: 0,
```

Comment:
EmailInput debounces state updates. Had to change this to get the tests to pass.